porch/pkg/git/draft.go

```diff
@@ -112,13 +114,22 @@ func (d *gitPackageDraft) Close(ctx context.Context) (repository.PackageRevision
 
 func (r *gitRepository) closeDraft(ctx context.Context, d *gitPackageDraft) (*gitPackageRevision, error) {
 	refSpecs := newPushRefSpecBuilder()
-	draftBranch := createDraftName(d.path, d.revision)
-	proposedBranch := createProposedName(d.path, d.revision)
+	draftBranch := createDraftName(d.path, d.description)
```

Comment:
tip: If you define a `type WorkspaceName string` or similar, the go compiler should identifying anywhere we need to pass a WorkspaceName.  (I don't see any problems, just that we should get the compiler to help us!)

Reply:
Done